pkg/client/unversioned/policy.go

```diff
@@ -65,12 +65,10 @@ func setPolicyDefaults(config *restclient.Config) error {
 	if config.UserAgent == "" {
 		config.UserAgent = restclient.DefaultKubernetesUserAgent()
 	}
-	// TODO: Unconditionally set the config.Version, until we fix the config.
-	//if config.Version == "" {
-	copyGroupVersion := g.GroupVersion
-	config.GroupVersion = &copyGroupVersion
-	//}
-
+	if config.GroupVersion == nil || !isEnabledVersion(policy.GroupName, config.GroupVersion) {
```

Comment:
We already have a proposal to solve that in https://github.com/kubernetes/kubernetes/pull/26044